index.js

```diff
@@ -73,6 +73,17 @@ Object.defineProperty(exports, 'urlencoded', {
   get: createParserGetter('urlencoded')
 })
 
+/**
+ * Generic parser used to build parsers.
+ * @public
+ */
+
+Object.defineProperty(exports, 'generic', {
+  configurable: true,
+  enumerable: true,
+  get: createParserGetter('generic')
+})
```

Comment:
I think the generic parser should not be exported for now, allowing us to refine its design without risking breaking user code.